### ISSUE TITLE
Replace historie with StoryBook

### ISSUE THREAD

**Issue by zoey-kaiser** [sidestream-tech/sidebase#65]
### Describe the feature

The newest version of Storybook allows you to drop webpack and only use vite. As we are currently having issue upgrading to newer release candidates, due to an issue with historie it may make sense to drop historie for now and replace it with storybook.

https://twitter.com/storybookjs/status/1582023943098990592

**Comment by BracketJohn** [comment]
Histoire: https://histoire.dev/
Stroybook: storybook.js.org/

**Comment by david8z** [comment]
I agree with this point, I would love to use Histoire as I love its simplicity, but every time I try to configure it, I find new things that don't work as expected.

**Comment by JPustkuchen** [comment]
Yep, Storybook seems the more stable choice and is more widely used.

The tailwind receipt might also be interesting here: https://storybook.js.org/recipes/tailwindcss
